### PR TITLE
Nm dol repo

### PR DESCRIPTION
**Pull request by marneneha**
added changes for "how to launch" in readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,5 +8,22 @@ planner : MaxlPlanner (devloped at Mowito)
...
+ROS Melodic/Neotic (Ros1)<br/>
+Rviz (present by default if not [refer](http://wiki.ros.org/rviz/UserGuide))<br/>
+For further ROS package dependencies [refer](https://docs.google.com/document/d/1ax2jxiBoe7LoIBfpz3iLcvHPjz6RKT2QiCndPhsWyjA/edit)
+#### Repository requirement
+For running the simulation you must have Maxl Planner install which is mention above<br/>
+For installing Maxl planner [refer](https://docs.google.com/document/d/1ax2jxiBoe7LoIBfpz3iLcvHPjz6RKT2QiCndPhsWyjA/edit)
+For further information on work visit [Mowito](https://mowito.in/)
+### Launch simulation
+After building and sourcing the repo on terminal type follwing commands 
+1. ```roslaunch rosbot_description rosbot_nav_with_map.launch <br/>```
```
Comment (Divyanshu-Sahu) [inline README.md]: "br/" is coming in the main readme as well.

File: README.md [hunk 2]
```diff
@@ -8,5 +8,22 @@ planner : MaxlPlanner (devloped at Mowito)
 Keyboard innput can be given at run time<br/> 
 robot will update path taking obstacle avoidance in account as goal is changed by keys i , l j to move it forword backword rigth left 
 
+# How to launch
+### System requirement
+Ubuntu 18/20<br/>
+ROS Melodic/Neotic (Ros1)<br/>
```
Comment (Divyanshu-Sahu) [inline README.md]: ROS1 instead of Ros1

File: README.md [hunk 3]
```diff
@@ -8,5 +8,22 @@ planner : MaxlPlanner (devloped at Mowito)
...
+Rviz (present by default if not [refer](http://wiki.ros.org/rviz/UserGuide))<br/>
+For further ROS package dependencies [refer](https://docs.google.com/document/d/1ax2jxiBoe7LoIBfpz3iLcvHPjz6RKT2QiCndPhsWyjA/edit)
```
Comment (Divyanshu-Sahu) [inline README.md]: ..............further information on ROS package dependencies

File: README.md [hunk 4]
```diff
@@ -8,5 +8,22 @@ planner : MaxlPlanner (devloped at Mowito)
...
+#### Repository requirement
+For running the simulation you must have Maxl Planner install which is mention above<br/>
```
Comment (Divyanshu-Sahu) [inline README.md]: ..........you must have the mw_maxl_planner repo installed. More information can be found on this link (add URL to the link)

File: README.md [hunk 5]
```diff
@@ -8,5 +8,22 @@ planner : MaxlPlanner (devloped at Mowito)
...
+For installing Maxl planner [refer](https://docs.google.com/document/d/1ax2jxiBoe7LoIBfpz3iLcvHPjz6RKT2QiCndPhsWyjA/edit)
```
Comment (Divyanshu-Sahu) [inline README.md]: NOTE: This is the wrong link. This google doc is only to be given to an employee not the customer. Customer have to go through read the docs to get their information. Please remove this link ASAP and update the link with read the docs link. Also, please be more descriptive in readme.

File: README.md [hunk 6]
```diff
@@ -8,5 +8,22 @@ planner : MaxlPlanner (devloped at Mowito)
 Keyboard innput can be given at run time<br/> 
 robot will update path taking obstacle avoidance in account as goal is changed by keys i , l j to move it forword backword rigth left 
 
+# How to launch
+### System requirement
+Ubuntu 18/20<br/>
+ROS Melodic/Neotic (Ros1)<br/>
+Rviz (present by default if not [refer](http://wiki.ros.org/rviz/UserGuide))<br/>
```
Comment (Divyanshu-Sahu) [inline README.md]: ....refer to the link (add URL to the word link)

Do the same for rest of the places as well

**Review by puru07 — APPROVED** [review]

**Pull request merged**